### DISCUSSION TITLE
tailwing in my project not getting installed as i am using parcel fro react

### DISCUSSION THREAD

**Discussion by mamta-vyas** [tailwindlabs/tailwindcss#14075] [Help]
why i am getting this following error

Unknown at rule @tailwind unknownAtrules (Ln1, Col 1)

**Comment by wongjn** [comment]
Assuming this is in VSCode, it is because VSCode's built-in CSS validation feature does not understand Tailwind-specific CSS syntax.

Make the warning go away, you could look at installing the [Tailwind CSS IntelliSense VSCode plugin](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss) and then:

> ### files.associations
> Use the files.associations setting to tell VS Code to always open `.css` files in Tailwind CSS mode:
>
> ```json
> "files.associations": {
>   "*.css": "tailwindcss"
> }

Otherwise, you could set `css.lint.unknownAtRules` VSCode setting to `ignore`.
